src/Selenium2Library/keywords/browsermanagement.py

```diff
@@ -62,6 +65,32 @@ def close_browser(self):
             self.browsers.close()
 
     @keyword
+    def delete_all_cookies_across_all_domains(self):
```

Comment:
Cookie keywords belongs to [CookieKeywords](https://github.com/robotframework/Selenium2Library/blob/master/src/Selenium2Library/keywords/cookie.py). Would it be good idea to incorporate the code with existing `Delete All Cookies`keyword? There is some backwards incompatible issue worries but that should not be problem.